File: src/Umbraco.Cms.Api.Management/ViewModels/AuditLogs/AuditLogWithUsernameResponseModel.cs
using Umbraco.Cms.Core.Models;

namespace Umbraco.Cms.Api.Management.ViewModels.AuditLogs;

public class AuditLogWithUsernameResponseModel : AuditLogBaseModel
{
    public string? UserName { get; set; }

    public string[]? UserAvatars { get; set; }
}

File: src/Umbraco.Web.UI.Client/libs/backend-api/src/models/AuditLogWithUsernameResponseModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { AuditLogBaseModel } from './AuditLogBaseModel';

export type AuditLogWithUsernameResponseModel = (AuditLogBaseModel & {
userName?: string | null;
userAvatars?: Array<string> | null;
});
